godot-core/src/bind.rs

```diff
@@ -24,6 +24,66 @@ use crate::obj::GodotClass;
 ///
 /// Do not call any of these methods directly -- they are an interface to Godot. Functionality
 /// described here is available through other means (e.g. `init` via `Gd::new_default`).
+///
+/// It is not enough to impl GodotExt to be registered in Godot, for this you should look at [ExtensionLibrary](crate::init::ExtensionLibrary)
+///
+/// # Examples
+///
+/// ## Basic Reference
+///
+/// ```
+/// use godot::prelude::*;
+///
+/// #[derive(GodotClass)]
+/// struct MyRef;
+///
+/// #[godot_api]
+/// impl MyRef {
+/// #[func]
+/// pub fn hello_world(&mut self) {
+///      godot_print!("Hello World!")
+///    }
+/// }
+///
+/// #[godot_api]
+/// impl GodotExt for MyRef {
+///     fn init(_: Base<RefCounted>) -> Self {
+///         MyRef
+///     }
+/// }
+/// ```
+///
+/// The following example allows to reference MyStruct in gdscript by calling for instance
+/// `MyStruct.new().hello_world()`.
+///
+///
+/// Note that you have to implement init otherwise you won't be able to call new or any
+/// other methods from gdscript
```

Comment:
Missing period here, and some other places too.

Reply:
thanks!